docs/UserGuide.md

```diff
@@ -131,19 +143,35 @@ Examples:
 * `find alex david` returns `Alex Yeoh`, `David Li`<br>
   ![result for 'find alex david'](images/findAlexDavidResult.png)
 
+### Locating persons by phone number: `findnum`
+
+Finds persons whose phone number matches the given keywords.
+
+Format: `find PHONE_NUMBER`
+
+* The search is strictly numeric. e.g `83647382` will only match `83647382`.
+* The order of the keywords matters. e.g. `83647382` will not match `83467328`
+* Only the full number is searched. eg. `83647382` will not match `83641001`
+
+Examples:
+* `find 83647382` returns `john` and `John Doe`
+* `find 91739272` returns `David Li`<br>
```

Comment:
```suggestion
* `findnum 83647382` returns `john` and `John Doe`
* `findnum 91739272` returns `David Li`<br>
```

Reply:
Should return result be changed too? A number should be unique and couldn't possibly be matched to both `john` and `John Doe`